packages/cli/src/index.ts

```diff
@@ -11,27 +11,31 @@
  *
  */
 
-import {Imperative} from "@zowe/imperative";
+import { Imperative } from "@zowe/imperative";
 
 /**
  * Initialize with imperative; init() will read package.json and load
  * our configurationModule, src/imperative/configuration.ts
  */
-Imperative.init().then(() => {
-
-    // if we initialize, we can use our app logging space for debug messages and other logging
-    Imperative.api.appLogger.debug("Initialized successfully");
-    Imperative.api.additionalLogger("another").debug("First additional logger configured");
-    Imperative.api.additionalLogger("yetAnother").debug("Second additional logger configured");
-    // Imperative.api.defaultConsole.debug(inspect(Imperative.api));
-
-    // have imperative parse command arguments can give our handler control if everything is ok
-    Imperative.parse();
-
-// handle any potential errors
-}).catch((e: Error) => {
-
-    // since imperative had an initialization error, we cannot use API methods
-    Imperative.console.fatal(require("util").inspect(e));
-});
-
+Imperative.init()
+    .then(() => {
+        // if we initialize, we can use our app logging space for debug messages and other logging
+        Imperative.api.appLogger.debug("Initialized successfully");
+        Imperative.api
+            .additionalLogger("another")
+            .debug("First additional logger configured");
+        Imperative.api
+            .additionalLogger("yetAnother")
+            .debug("Second additional logger configured");
+
+        // have imperative parse command arguments can give our handler control if everything is ok
+        Imperative.parse();
+
+        // handle any potential errors
+    })
+    .catch((e: Error) => {
+        // since imperative had an initialization error, we cannot use API methods
+        Imperative.console.fatal(require("util").inspect(e));
+    });
```

Comment:
This block of code can likely be removed. It came from the SCS plugin when we copied that repo, but not sure why it was there either. The only thing we need to do in this file is export our credential manager and initializing Imperative shouldn't be necessary. For an example of a simpler `index.ts` file: https://github.com/zowe/zowe-cli-sample-plugin/blob/master/src/index.ts